snuba/admin/clickhouse/predefined_querylog_queries.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from snuba.admin.clickhouse.common import PreDefinedQuery
+from snuba.utils.registered_class import RegisteredClass
+
+
+class QuerylogQuery(PreDefinedQuery, metaclass=RegisteredClass):
+    @classmethod
+    def config_key(cls) -> str:
+        return cls.__name__
+
+
+class QueryByID(QuerylogQuery):
+    """Find a query by its ID"""
+
+    sql = """
+    SELECT <fields>
+    FROM querylog_local
+    WHERE has(clickhouse_queries.query_id, '<UUID>')
+    AND timestamp > (now() - 60)
+    AND timestamp < toDateTime('2022-11-18T19:20:00')
```

Comment:
Any particular reason why you are using a specific end timestamp instead of now()

Reply:
Just to show how to do that. Sometimes you want to isolate a specific time window.